code/__DEFINES/DNA.dm

```diff
@@ -81,22 +81,17 @@
 #define AGENDER 14
 /// Do not draw eyes or eyeless overlay
 #define NOEYESPRITES 15
-/// Used for determining which wounds are applicable to this species.
-/// if we have flesh (can suffer slash/piercing/burn wounds, requires they don't have NOBLOOD)
-#define HAS_FLESH 16
-/// if we have bones (can suffer bone wounds)
-#define HAS_BONE 17
 ///If we have a limb-specific overlay sprite
-#define HAS_MARKINGS 18
+#define HAS_MARKINGS 16
```

Comment:
Forgot to update the numbers? HAS_FLESH and HAS_MARKINGS will be the same define now (aswell as all the others that share the same value)

Reply:
HAS_FLESH and HAS_BONE have been removed, neither of them make sense as species traits now.

Reply:
I double checked, and every species trait has been updated properly.